web/docs/try/quickstart.md

```diff
@@ -0,0 +1,420 @@
+---
+sidebar_position: 1
+---
+
+# Quickstart
+
+This guide illustrates how you can use VAST from the command line interface.
+We're assuming that you have [installed VAST](/docs/setup/install) and that
+you have a `vast` binary in your path.
+
+## Start a VAST node
+
+Let's spin up a VAST node:
+
+```bash
+vast start
+```
+
+```
+[20:05:54.138] VAST (v3.0.0-rc1-2-gcc393580f1) is listening on 127.0.0.1:42000
+[20:05:54.144] index-11 finished initializing and is ready to accept queries
+```
+
+All other commands aside from `start` initiate a connection to a VAST node. For
```

Comment:
This is slightly false as `vast version` and `vast infer` do not connect to a node at all, and neither do commands run via `vast --node <command>`.